spine-c/src/spine/Bone.c

```diff
@@ -209,9 +206,9 @@ float spBone_getWorldScaleY (spBone* self) {
 }
 
 void spBone_worldToLocal (spBone* self, float worldX, float worldY, float* localX, float* localY) {
-	float x = worldX - self->worldX, y = worldY - self->worldY;
 	float a = self->a, b = self->b, c = self->c, d = self->d;
 	float invDet = 1 / (a * d - b * c);
+	float x = worldX - self->worldX, y = worldY - self->worldY;
 	*localX = (x * d * invDet - y * b * invDet);
 	*localY = (y * a * invDet - x * c * invDet);
 }
```

Comment:
Got it. Committed at https://github.com/EsotericSoftware/spine-runtimes/pull/619/commits/7a3e322d9f597c4df6e781652011a0e11a3824eb.